packages/components/checkbox/src/components/osds-checkbox/interfaces/attributes.ts

```diff
@@ -1,6 +1,8 @@
+import type { OdsGenericFormFieldAttribute } from '@ovhcloud/ods-common-core';
+
 type OdsCheckboxAttributeCbk = ({ checked, value }: { checked: boolean, value: HTMLInputElement['value'] }) => Promise<void>;
 
-interface OdsCheckboxAttribute {
+interface OdsCheckboxAttribute extends OdsGenericFormFieldAttribute {
```

Comment:
Same here, Form field component exists, so maybe we should not use the same naming

Reply:
Name changed